src/service.js

```diff
@@ -83,20 +88,27 @@ LS2Request.prototype.send = function() {
 		}
 		if(!self.subscribe) {
 			self.cancel();
+		} else if(self.ts && refs[self.ts]) {
```

Comment:
Sorry, didn't see the comment above. 
Correct me if I'm wrong, but removing ref after the first callback defeats the purpose of subscription as it will never receive the subsequent callbacks.
I can't think of a good use case where a user would want to unsubscribe at any point, either. Normally, a user will subscribe to a value (usually by a getter APIs) and the setter function or any other related methods that would change the subscribed value would invoke a callback. This allows users to only fetch data once and rely on subscription afterwards. So in many cases, it shouldn't unsubscribe at all.

Another question I have is, does `PalmServiceBridge` provide a proper way to "unsubscribe"?